server/docker.yaml

```diff
@@ -0,0 +1,6 @@
+docker:
+    command: docker
+    init_command: \{command} .. (optional or default)
+    start_command: \{command} run {image}
+    stop_command: \{command} (optional or kill default)
+    pull_command: \{command} pull {image} {url}
```

Comment:
The server yamls need newlines.

Reply:
Added the newlines to the server yamls as well